### PR TITLE
Added management command to check on and cancel SMS rules

### PR DESCRIPTION
**Pull request by orangejenny**
Checking Daniel's script into source control.

### CODE REVIEW

File: corehq/messaging/management/commands/manage_rule_run.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        if rule.domain != domain:
+            raise CommandError("Domain '%s' does not match rule's domain '%s'" % (domain, rule.domain))
+
+        if rule.workflow != AutomaticUpdateRule.WORKFLOW_SCHEDULING:
+            raise CommandError("Expected the rule to be a messaging rule")
+
+        return rule
+
+    def print_status(self, rule):
+        rule = AutomaticUpdateRule.objects.get(id=rule.id)
```
Comment (mkangia) [inline corehq/messaging/management/commands/manage_rule_run.py]: why did we fetch it again with id now?

File: corehq/messaging/management/commands/manage_rule_run.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            self.print_status(rule)
+
+            if cancel:
+                confirm = input("Are you sure you want to cancel this rule?  This is NOT a dry run. y/N?")
+                if confirm == "y":
+                    msg = MessagingRuleProgressHelper(rule.id)
+                    if msg.is_canceled():
+                        print("already canceled")
+                    else:
+                        print("canceled rule", rule_id)
```
Comment (mkangia) [inline corehq/messaging/management/commands/manage_rule_run.py]: i think this should say `cancelling rule` and then `canceled` should be after the msg has actually been cancelled

**Review by mkangia — APPROVED** [review]

**Review by millerdev — APPROVED** [review]

File: corehq/messaging/management/commands/manage_rule_run.py (hunk 1 above)
Comment (orangejenny) [reply corehq/messaging/management/commands/manage_rule_run.py]: Oops, I copied this from another script. Will fix.

File: corehq/messaging/management/commands/manage_rule_run.py (hunk 2 above)
Comment (orangejenny) [reply corehq/messaging/management/commands/manage_rule_run.py]: Moved message after the cancel call.

**Comment by orangejenny** [comment]
Killed build, as there's no test coverage of this.

**Pull request merged**